packages/storybook/src/schematics/migrate-defaults-5-to-6/migrate-defaults-5-to-6.ts

```diff
@@ -0,0 +1,274 @@
+              .uiFramework,
+          configFolder:
+            workspaceJson.projects[project].targets.storybook.options.config
+              .configFolder,
+        };
+      }
+      if (
+        workspaceJson.projects[project].architect &&
+        workspaceJson.projects[project].architect.storybook
+      ) {
+        return {
+          name: project,
+          uiFramework:
+            workspaceJson.projects[project].architect.storybook.options
+              .uiFramework,
+          configFolder:
+            workspaceJson.projects[project].architect.storybook.options.config
+              .configFolder,
+        };
+      }
+    });
+
+    return chain(
+      projectsThatHaveStorybookConfiguration
+        .filter(
+          (projectWithStorybook) =>
+            projectWithStorybook && projectWithStorybook.configFolder
+        )
+        .map((projectWithStorybook) => {
+          return chain([
+            migrateStorybookInstance(
+              keepOld,
+              projectWithStorybook.name,
+              projectWithStorybook.uiFramework,
+              projectWithStorybook.configFolder
+            ),
+          ]);
+        })
+    );
+  };
+}
+
+export function migrateStorybookInstance(
+  keepOld: boolean,
+  projectName: string,
+  uiFramework: string,
+  configFolder?: string
+): Rule {
+  return chain([
+    migrateProjectLevelStorybookInstance(
+      keepOld,
+      projectName,
+      uiFramework,
+      configFolder
+    ),
+  ]);
+}
+
+function upgradeStorybookPackagesInPackageJson(): Rule {
+  /**
+   * Should upgrade all @storybook/* packages in package.json
+   */
+  let needsInstall = false;
+
+  const maybeUpdateVersion = updateJsonInTree('package.json', (json) => {
+    json.dependencies = json.dependencies || {};
+    json.devDependencies = json.devDependencies || {};
+
+    const storybookPackages = [
+      '@storybook/angular',
+      '@storybook/react',
+      '@storybook/addon-knobs',
+    ];
+
+    storybookPackages.forEach((storybookPackageName) => {
+      if (json.dependencies[storybookPackageName]) {
+        const version = checkAndCleanWithSemver(
+          storybookPackageName,
+          json.dependencies[storybookPackageName]
+        );
+        if (lte(version, '6.0.0')) {
+          json.dependencies[storybookPackageName] = '^6.1.11';
+          needsInstall = true;
+        }
+      }
+      if (json.devDependencies[storybookPackageName]) {
+        const version = checkAndCleanWithSemver(
+          storybookPackageName,
+          json.devDependencies[storybookPackageName]
+        );
+        if (lte(version, '6.0.0')) {
+          json.devDependencies[storybookPackageName] = '^6.1.11';
+          needsInstall = true;
+        }
+      }
+    });
+
+    return json;
+  });
+
+  return (tree: Tree, context: SchematicContext) => {
+    return chain([
+      maybeUpdateVersion,
+      formatFiles(),
+      addInstallTask({ skipInstall: !needsInstall }),
```

Comment:
🤔 interesting. I thought this piece is missing. In my case it didn't trigger an install during the migration. I had to manually run `yarn install` so maybe something is not working with this one.